blooket.py

```
@@ -0,0 +1,20 @@
+# Credit to bigboy-baby (https://github.com/bigboy-baby/Blooket-Pin-Crackers)
+import requests, random, string, threading
+from datetime import datetime
+print("Credit to bigboy-baby on GitHub (https://github.com/bigboy-baby/Blooket-Pin-Crackers")
+
+def crack():
+  while True:
+    game = ('').join(random.choices(string.digits, k=6))
+    r = requests.get(f"https://fb.blooket.com/c/firebase/id?id={game}")
+    res = r.json()
+    if res["success"] == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if res["success"]:` for truth checks

Replace with `res["success"]`

(E712)